### PR TITLE
small runtime fixes

### PR DESCRIPTION
**Pull request by toomykins**
[bugfix]
small bug/runtime fixes

### CODE REVIEW

File: code/modules/cmc/crew.dm [hunk 1]
```diff
@@ -336,16 +336,15 @@ HOLOMAP PROCS
 		user.client.screen -= holomap_tooltips[uid]
 
 	if(holomap_images[uid])
-		holomap_images[uid].len = 0
```
Comment (DamianX) [inline code/modules/cmc/crew.dm]: why?

File: code/modules/virus2/helpers.dm [hunk 2]
```diff
@@ -112,6 +112,8 @@ proc/airborne_can_reach(turf/source, turf/target, var/radius=5)
 	return FALSE
 
 /proc/attempt_colony(var/atom/A, var/datum/disease2/disease/D,var/info)
+	if(!D || !A)
```
Comment (DamianX) [inline code/modules/virus2/helpers.dm]: what is calling this with either argument null?

Comment (toomykins) [reply code/modules/virus2/helpers.dm]: /obj/item/weapon/virusdish/attack_self was calling it. i fixed it, but i am scared someone will fuck it up again.

File: code/modules/cmc/crew.dm (hunk 1 above)
Comment (toomykins) [reply code/modules/cmc/crew.dm]: found a runtime [18:09:50] Runtime in crew.dm,387: Cannot modify null.len.
couldnt find the exact reason why it was null at the time, but figured making it a new empty list would give the same effect as clearing the list. i did it with the other .len = 0 instances in the file too for ease of reading. can be reverted if ye want

File: code/modules/virus2/helpers.dm (hunk 2 above)
Comment (toomykins) [reply code/modules/virus2/helpers.dm]: via /obj/item/weapon/virusdish/infection_attempt, that is.

File: code/modules/cmc/crew.dm (hunk 1 above)
Comment (DamianX) [reply code/modules/cmc/crew.dm]: It's checked on line 338 for a non-null value, something else must be going on.

Comment (DamianX) [reply code/modules/cmc/crew.dm]: Nevermind, got the line numbers mixed up. This isn't necessarily horrible but you should figure out if `holomap_tooltips[uid]` being `null` at line 387 is actually something that's not supposed to happen

File: code/modules/virus2/helpers.dm (hunk 2 above)
Comment (DamianX) [reply code/modules/virus2/helpers.dm]: Isn't a runtime in here the only thing that led you to discover this bug? If you add these checks you will not know if someone fucked up again, if I'm understanding correctly.

File: code/modules/cmc/crew.dm (hunk 1 above)
Comment (toomykins) [reply code/modules/cmc/crew.dm]: it's set to null in /deactivate, i mostly changed that line for consistency. it has the same effect, at least.

File: code/modules/virus2/helpers.dm (hunk 2 above)
Comment (toomykins) [reply code/modules/virus2/helpers.dm]: If someone in the future commits code that isn't allowed to function due to a null check, it's just proof they didn't test it.
I'll get rid of it though

File: code/modules/cmc/crew.dm (hunk 1 above)
Comment (DamianX) [reply code/modules/cmc/crew.dm]: @PaulRitter you coded this, what the fuck is going on

Comment (PaulRitter) [reply code/modules/cmc/crew.dm]: ~~wait what the fuck. how the hell is this runtiming, shouldn't if(holomap_images[uid]) prevent exactly this?~~ nevermind

Comment (PaulRitter) [reply code/modules/cmc/crew.dm]: this genuinly should never happen cause before this gets called, `initializeUser` should set it. `deactive` clears it and disconnects the user from the holomap

Comment (PaulRitter) [reply code/modules/cmc/crew.dm]: but yeah the fix is fine

Comment (toomykins) [reply code/modules/cmc/crew.dm]: initializeuser is only called when you click the pc, not when you're already in it.
you can still have the interface of the computer open even when the /deactivate function sets shit to null
at least thats my explanation

**Comment by DamianX** [comment]
you got conflicts

**Pull request merged**